pkgs/development/libraries/cairo/default.nix

```
@@ -49,8 +49,8 @@ in {
     (fetchpatch {
       name = "CVE-2020-35492.patch";
       includes = [ "src/cairo-image-compositor.c" ];
-      url = "https://github.com/freedesktop/cairo/commit/78266cc8c0f7a595cfe8f3b694bfb9bcc3700b38.patch";
-      sha256 = "048nzfz7rkgqb9xs0dfs56qdw7ckkxr87nbj3p0qziqdq4nb6wki";
+      url = "https://gitlab.freedesktop.org/cairo/cairo/commit/78266cc8c0f7a595cfe8f3b694bfb9bcc3700b38.patch";
```

The legacy routes [are still there for now](https://gitlab.com/gitlab-org/gitlab/-/merge_requests/121555) but we probably shouldn't use them anymore...

```suggestion
      url = "https://gitlab.freedesktop.org/cairo/cairo/-/commit/78266cc8c0f7a595cfe8f3b694bfb9bcc3700b38.patch";
```

---

updated the url, also updated the other urls in a separate commit to be a bit more consistent